### PR TITLE
Add Account Created Alert

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

- refactor name of csoAccountExist state to Status
- update csoAccountStatus to include exists and isNew
- add Account Created success alert to indicate account creation
- resolves [FLA-236](https://justice.gov.bc.ca/jira/browse/FLA-236)

csoAccountCreated

## Type of change

- [x] New feature (non-breaking change w`hich adds functionality)
- [x] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ const checkCSOAccountStatus = (
         const csoAccountExists = userDetails.accounts.some(
           element => element.type === "CSO"
         );
-        if (csoAccountExists) setCsoAccountExists(true);
+        if (csoAccountExists)
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:46]: do we care if it doesn't?

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.stories.storyshot [hunk 2]
```diff
@@ -597,3 +597,334 @@ exports[`Storyshots PackageConfirmation Mobile 1`] = `
...
+              <div
+                class="bluegrey-content"
+              >
+                <span
+                  class="content-style"
+                >
+                  <p>
+                    CSO account 
+                    <b>
+                      blah
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.stories.storyshot:827]: Yea but have you considered blah?

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.stories.storyshot [hunk 3]
```diff
@@ -597,3 +597,334 @@ exports[`Storyshots PackageConfirmation Mobile 1`] = `
...
+                    </b>
+                     is linked to your Basic BCeID account 
+                    <b>
+                      blah2
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.stories.storyshot:831]: Blah

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap [hunk 4]
```diff
@@ -296,3 +296,332 @@ exports[`PackageConfirmation Component Matches the snapshot 1`] = `
...
+              <div
+                class="bluegrey-content"
+              >
+                <span
+                  class="content-style"
+                >
+                  <p>
+                    CSO account 
+                    <b>
+                      blah
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap:524]: Blah blah black sheep.

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap [hunk 5]
```diff
@@ -296,3 +296,332 @@ exports[`PackageConfirmation Component Matches the snapshot 1`] = `
...
+                    </b>
+                     is linked to your Basic BCeID account 
+                    <b>
+                      blah2
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap:528]: Blahb Ross.

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap [hunk 6]
```diff
@@ -296,3 +296,332 @@ exports[`PackageConfirmation Component Matches the snapshot 1`] = `
...
+              <path
+                d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
+              />
+            </svg>
+          </div>
+        </div>
+        <div
+          class="display-element"
+        >
+          CSO Account created successfully.
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap:336]: Your CSO Account?

File: src/frontend/efiling-frontend/src/components/page/home/Home.js (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-frontend/src/components/page/home/Home.js:46]: if it does not exist, then the current state will handle it for now. later when we know our error handling approach, we will do some custom stuff :)

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js [hunk 7]
```diff
@@ -105,5 +118,8 @@ export default function PackageConfirmation({
 PackageConfirmation.propTypes = {
   packageConfirmation: PropTypes.shape({
     confirmationPopup: propTypes.confirmationPopup
+  }).isRequired,
+  csoAccountStatus: PropTypes.shape({
+    isNew: PropTypes.bool
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js:123]: no `exists` property here?

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap (hunk 6 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-frontend/src/components/page/package-confirmation/__snapshots__/PackageConfirmation.test.js.snap:336]: I like this, can check with julia and then make the change if she's good with it

File: src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js (hunk 7 above)
Comment (adodge-bcg) [reply src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js:123]: Should we pass the exists? Currently, the only thing we check is if the account is new, not if the account exists

Comment (sdevalapurkar-bcgov) [reply src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js:123]: hmm I see, yeah I guess the assumption is that when we arrive on this page, cso account already exists.
so no need to pass in, cool

**Pull request merged**